db/migrate/20170210151154_view_as_mappable_measurements.rb

```diff
@@ -0,0 +1,26 @@
+class ViewAsMappableMeasurements < ActiveRecord::Migration[5.0]
+  def up
+    execute <<-SQL
+      create materialized view mappable_measurements
+      as select
+        id,
+        created_at,
+        updated_at,
+        device_id,
+        (payload->>'captured_at')::timestamptz at time zone 'UTC' as captured_at,
```

Comment:
I am curious that what would happen when `payload->>'captured_at'` is null value. A`NULL`?